iphone/cli/hooks/frameworks.js

```diff
@@ -55,7 +61,12 @@ class FrameworkManager {
 			post: (builder, callback) => {
 				this._logger.trace('Starting third-party framework detection');
 				this._builder = builder;
-				this.detectFrameworks().then(callback, callback);
+				this.detectFrameworks().then(callback, e => {
+					if (this._cli.argv.platform === 'ios') {
```

Comment:
why spit out the error stack trace only on iOS? Won't this always be true? Wouldn't we spit out the full error before?

Reply:
Well, it's kinda messy (surprise!). This has been an issue on both platforms until Josh refactored the whole Android build for the Gradle support. Now the Android build manually spits out the error [here](https://github.com/appcelerator/titanium_mobile/blob/41ebeecedf3661685bff8ad6626d30295aa488e6/android/cli/commands/_build.js#L1575). The iOS build on the other hand has no special handling and relies on the default error handling in the common builder [here](https://github.com/appcelerator/titanium_mobile/blob/master/cli/commands/build.js#L332) which only prints out the error message.